frontend/ios/DroidKaigi 2019/Scenes/Session/TagsCollectionViewCell.swift

```diff
@@ -93,7 +93,6 @@ class TagsCollectionViewCell: UICollectionViewCell, Reusable {
         addSubview(label)
         label.snp.makeConstraints {
             $0.edges.equalToSuperview()
-            $0.width.lessThanOrEqualTo(230)
```

Comment:
Doesn't it work if we make the right constraint of the label fit to the right of the cell?

Reply:
Gotcha. Thank you so much!